src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -573,6 +617,31 @@ export class StudyViewPageStore {
         }
     }
 
+    //Save any shared custom data to user profile if user login after page is loaded
+    public async saveSharedCustomDataToUserProfile(
```

Comment:
kinda minor but this method doesn't return anything so probably doesn't need to by async.